drivers/ieee802154/ieee802154_rf2xx_regs.h

```diff
@@ -0,0 +1,307 @@
+/* ieee802154_rf2xx_regs.h - ATMEL RF2XX transceicer registers */
+
+/*
+ * Copyright (c) 2019 Gerson Fernando Budke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_IEEE802154_IEEE802154_RF2XX_REGS_H_
+#define ZEPHYR_DRIVERS_IEEE802154_IEEE802154_RF2XX_REGS_H_
+
+/*- Definitions ------------------------------------------------------------*/
+#define AES_BLOCK_SIZE                      16
+#define AES_CORE_CYCLE_TIME                 24  /* us */
+#define RANDOM_NUMBER_UPDATE_INTERVAL       1   /* us */
+
+/*- Types ------------------------------------------------------------------*/
```

Comment:
About the naming, add RF2XX_ prefix to all macros here. It is then clearer when grepping etc...

Reply:
Done.

Reply:
Done.